File: C#/Domain/DTO/GravarFotoOsDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Cebi.Atendimento.Domain.DTO
{
    public class GravarFotoOsDTO
    {
        public string EquipeId { get; set; }
        public string EquipamentoId { get; set; }

        public int EventoId { get; set; }
        public int OrdemServicoId { get; set; }
        public int Sequencia { get; set; }
        public string XmlFoto { get; set; }
    }
}

File: Java(Transcrição do C#)/Model/DTO/GravarFotoOsDTO.java
package amostras.domain.entities;


{
    public class GravarFotoOsDTO
    {
        private String equipeId ;
        private String equipamentoId ;
        private Integer eventoId ;
        private Integer ordemServicoId ;
        private Integer sequencia ;
        private String xmlFoto ;

        public String getEquipeId() {
            return equipeId;
        }

        public void setEquipeId(String equipeId) {
            this.equipeId = equipeId;
        }

        public String getEquipamentoId() {
            return equipamentoId;
        }

        public void setEquipamentoId(String equipamentoId) {
            this.equipamentoId = equipamentoId
        }

        public Integer getEventoId() {
            return eventoId;
        }

        public void setEventoId(Integer eventoId) {
            this.eventoId = eventoId;
        }

        public Integer getOrdemServicoId() {
            return ordemServicoId;
        }

        public void setOrdemServicoId(Integer ordemServicoId) 
        {
            this.ordemServicoId = ordemServicoId;
        }

        public Integer getSequencia() {
            return sequencia;
        }

        public void setSequencia(Integer sequencia)
        {
            this.sequencia = sequencia;
        }

        public String getXmlFoto() {
            return xmlFoto;
        }

        public void setXmlFoto(String xmlFoto) 
        {
            this.xmlFoto = xmlFoto;
        }
    }
}
